### PR TITLE
load_ts_file 

### PR DESCRIPTION
**Pull request by aa25desh**
Add `load_ts_file()` function to load .ts files into Julia, load .ts file from hard drive and return IndexedTable without intermediate step of writing .csv file

### CODE REVIEW

File: src/datapath.jl [hunk 1]
```diff
@@ -79,6 +79,17 @@ function TSdataset(filepath::String)
     end
 end
 
+function load_ts_file(fpath)
+    data = open(fpath) do input
+          readuntil(input, "@data")
```
Comment (mloning) [inline src/datapath.jl:90]: It's good practice and coding habit to add 
* docstrings describing what the function does,
* short in-line comments describing what the different parts of the function do, e.g. the regex expression in line 88) and `readuntil()` function call in line 84.

File: Benchmark.jl [hunk 2]
```diff
@@ -0,0 +1,12 @@
+using MLJTime
+
+const univariate_data = univariate_datasets
+
+TSdataset(univariate_data)
+
+
+function Benchmake_all(algorithms)
```
Comment (mloning) [inline Benchmark.jl]: I wouldn't include unfinished functions in a PR

**Review by mloning — COMMENTED** [review]
Hi @aa25desh, 

Looks good, I left two comments on general coding practices!

File: Benchmark.jl (hunk 2 above)
Comment (aa25desh) [reply Benchmark.jl]: Oh, That function is added by mistake!

File: src/datapath.jl (hunk 1 above)
Comment (aa25desh) [reply src/datapath.jl:90]: I will push doctoring soon:)

**Comment by aa25desh** [comment]
Closing in favour of #4

**Pull request closed without merging**